index.bs

```diff
@@ -1586,11 +1586,10 @@ or otherwise return null.
 </table>
 
 <h2 id=url>
-A URL for Blob and File reference</h2>
+A URL for Blob, File and MediaSource reference</h2>
```

Comment:
Let's remove File (below too). We generally don't mention subclasses specifically.